lua/entities/gmod_wire_expression2/core/egpfunctions.lua

```diff
@@ -1028,18 +988,20 @@ end
 
 __e2setcost(20)
 e2function vector wirelink:egpToWorld( vector2 pos )
-	if not EGP:ValidEGP( this ) then return Vector(0,0,0) end
+	if not EGP:ValidEGP( this ) then return { 0, 0, 0 } end
```

Comment:
I would have assumed that if you weren't allowed to return a Vector, then someone would've noticed a very long time ago. If returning a Vector is allowed then these changes seem pointless - unless your plan is to also refactor all E2 functions to not return Vectors.